### DISCUSSION TITLE
Let's meet in person at FOSDEM 2024! :beers: :fries: :waffle:

### DISCUSSION THREAD

**Discussion by jimmykarily** [kairos-io/kairos#2114] [General]
Hello Kairos community!

We will be at [FOSDEM](https://fosdem.org/2024/) next month (3-4 February) and we think it would be a nice opportunity to meet in person.

We were thinking about having lunch together on Saturday and for whomever can't make it for lunch, we can also try some drinks in the evening.

Saturday is probably the best day, because there may be people that want to travel after FOSDEM on Sunday.
We also have [our talk on Sunday at 11:45](https://fosdem.org/2024/schedule/event/fosdem-2024-2983-from-free-lunch-to-dog-fooding-a-culinary-journey-in-crafting-iac-for-kairos-testing-and-building/) (Testing and Continuous delivery devroom) so we'll have an opportunity to meet there too.

To be able to arrange and book a place, it would be nice to know beforehand who might join.
You don't have to be 100% sure but let us know if you think you can make it so we can count heads.
Please reply in a comment and we'll keep you up to date with whatever we plan.

We'll be happy to see you there!

**Comment by Itxaka** [comment]
impossible for me to attend although would really like to attend next year and catch up with you fine folks with more preparation beforehand :) :frikandel:
